lib/screens/settings_screen.dart

```diff
@@ -51,7 +54,46 @@ class _SettingsScreenState extends State<SettingsScreen> {
                 icon: const Icon(Icons.logout_outlined),
                 tooltip: "Log out",
               ),
-            )
+            ),
+             const Center(
+                  child: Text(
+                "contact developer",
+                style: TextStyle(fontSize: 23),
+              )),
+            ListTile(
+              leading: RaisedButton(
+                onPressed: () async =>
+                    await launch("https://wa.me/${"+79871852923"}?text=Hello"),
+                child: const Text('Open WhatsApp'),
+                color: Colors.black,
+              ),
+              trailing: RaisedButton(
+                onPressed: () async =>
+                    await launch("https://t.me/taasneemtoolba"),
+                child: const Text('Open Telegram'),
+                color: Colors.black,
+
+              ),
+            ),
+            ListTile(
+              title: const Text("Rate us"),
+              trailing: RatingBar.builder(
+                initialRating: 0,
+                minRating: 1,
+                direction: Axis.horizontal,
+                allowHalfRating: true,
+                itemCount: 5,
+                itemPadding: EdgeInsets.symmetric(horizontal: 4.0),
```

Comment:
"Prefer const with constant constructors" - dart analyzer.
Please make sure that Android Studio shows you Dart Analyzer's messages